InvenTree/report/templatetags/report.py

```diff
@@ -184,13 +225,17 @@ def part_image(part: Part):
     Raises:
         TypeError if provided part is not a Part instance
     """
-    if type(part) is Part:
-        img = part.image.name
+    if type(part) is not Part:
+        raise TypeError("part_image tag requires a Part instance")
```

Comment:
Suggestion: This error is not translated
```suggestion
        raise TypeError(_("part_image tag requires a Part instance"))
```